dokka-integration-tests/gradle/src/main/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt

```diff
@@ -14,7 +14,7 @@ open class AllSupportedTestedVersionsArgumentsProvider : TestedVersionsArguments
 
 object TestedVersions {
 
-    val LATEST = BuildVersions("7.6.2", "1.9.22")
+    val LATEST = BuildVersions("8.5", "2.0.0-Beta4")
```

Comment:
> latest Gradle version is 8.6, should we use it here?

Damn, time flies and Gradle ships fast. Updated it to 8.6

I know there's 2.0.0-Beta5 now, but I won't update to it due to #3533, let's do it after this PR is merged, it has enough hacks as it is 😅 

> should we also test on versions between 7.6 and 8.6, or you think just latest minor will be enough?

Also a good question. Testing on the latest minor versions has worked well so far, we haven't had any problems with it, but Gradle 8 seems to have more features in minor releases, so I don't know, it might be needed. I'd be ideal to test on all intermediate versions, of course, but then the execution time would skyrocket

Perhaps, more versions could be tested if the `isExhaustive` flag is enabled, but last time I checked it didn't work well, some other versions need to get adjusted

Reply:
> Damn, time flies and Gradle ships fast. Updated it to 8.6

And soon there will be 8.7 - several RCs already :)

> but Gradle 8 seems to have more features in minor releases

Yeah, that's why I thought about it. 8.x versions are implementing a lot of new features regarding CC and IP, so it would be nice to have, but not strictly required.
May be after IT will be CC friendly we could check if adding some of them is feasible or not.

Reply:
too soon: https://github.com/gradle/gradle/releases/tag/v8.7.0

Reply:
FYI, on current moment we blocked by: https://youtrack.jetbrains.com/issue/KT-66961/Early-access-to-gradle.rootProject-leads-to-an-exception
Fixed in 2.0.0-RC1 (checked that it works with 2.0.0-RC1-86 from dev repository)
To prepare for this, I've updated Kotlin version to `2.0.0-Beta5` and so fixed #3533 (https://github.com/Kotlin/dokka/pull/3516/commits/fe5f5b3590f9f8ccbb3a15286fe95308be502408 - should be rather safe, as we only use `BindingContext.get` afterwards)

Reply:
I've updated latest Gradle version for IT to 8.7 and Kotlin to 2.0.0-RC1